### ISSUE TITLE
Using onEditable while using jsoneditor in NextJS if throwing error. 

### ISSUE THREAD

**Issue by vindu25** [josdejong/jsoneditor#1112]
![image](https://user-images.githubusercontent.com/50862598/93584927-a7f4d200-f9c3-11ea-817e-f7f5b56712d4.png)

I'm using the Json react example code.
 
```
const Editor = dynamic(
  () => import('../src/components/JSONEditorReact'),
  { ssr: false }
);
```

```
<Editor
                text={editorText}
                mode={mode}
                modes={['tree', 'view', 'code']}
                indentation={4}
                onChangeText={onTextChange}
                onModeChange={onModeChange}
                templates={templates}
                onEditable={onEdit}
              />
```

```
const onEdit(node) = {
    console.log(node);
}
```

**Comment by josdejong** [comment]
Thanks for reporting Aravind. How can I reproduce this issue exactly?

**Comment by vindu25** [comment]
@josdejong Created a sample app here https://github.com/vindu25/jsoneditorfailure , if it fits feel free to add it as an example to the repo. The sample app is using next js

**Comment by josdejong** [comment]
Thanks for working out this example! I started it locally and the editor seems to just work. What triggers the error exactly? (or is this issue maybe solved in the just released `v9.1.1`?)

**Comment by vindu25** [comment]
> Thanks for working out this example! I started it locally and the editor seems to just work. What triggers the error exactly? (or is this issue maybe solved in the just released `v9.1.1`?)

@josdejong If you uncomment this `onEditable` , the editor will fail to render 
 with above exception https://github.com/vindu25/jsoneditorfailure/blob/master/pages/index.js#L72

**Comment by josdejong** [comment]
Ah, now I can reproduce the issue 👍 

The cause is that your implementation of `onEditable` returns `undefined`, whilst it is supposed to return either a `boolean` or an object, `{ field: boolean, value: boolean }`. See docs https://github.com/josdejong/jsoneditor/blob/develop/docs/api.md -> search for `onEditable`.

**Comment by josdejong** [comment]
I've improved the handling of `onEditable` to log a console error when the output of `onEditable` is invalid, see 58f460a. (Not yet published)

**Comment by vindu25** [comment]
> I've improved the handling of `onEditable` to log a console error when the output of `onEditable` is invalid, see [58f460a](https://github.com/josdejong/jsoneditor/commit/58f460a383b6e30501b5f28e2eeb0d9beb0b79d9). (Not yet published)

Awesome thank you very much for quick reply, this really helps.

**Comment by josdejong** [comment]
This is (finally) published in `v9.1.2`.
